modules/gdscript/gdscript.cpp

```diff
@@ -1363,6 +1363,8 @@ GDScript::GDScript() :
 
 		GDScriptLanguage::get_singleton()->script_list.add(&script_list);
 	}
+
+	path = vformat("script://%s.gd", get_instance_id());
```

Comment:
This technically should use `%d` since it's an `int` (well, it's and `ObjectID` but that is converted to `int`). Doesn't really make a difference though.

```suggestion
	path = vformat("script://%d.gd", get_instance_id());
```

Reply:
Done!